src/Sylius/Bundle/CoreBundle/Provider/SymfonyTranslatorResourceProvider.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Provider;
+
+use Sylius\Bundle\ThemeBundle\Translation\Provider\Resource\TranslatorResourceProviderInterface;
+use Sylius\Bundle\ThemeBundle\Translation\Resource\TranslationResource;
+use Sylius\Bundle\ThemeBundle\Translation\Resource\TranslationResourceInterface;
+
+/** @inner */
```

Comment:
```suggestion
/** @internal */
```
according to https://docs.sylius.com/en/1.9/book/organization/backward-compatibility-promise.html#code-covered